bentoml/yatai/db/stores/metadata.py

```diff
@@ -105,13 +105,20 @@ def _bento_orm_obj_to_pb(bento_obj, labels=None):
     bento_uri = BentoUri(
         uri=bento_obj.uri, type=BentoUri.StorageType.Value(bento_obj.uri_type)
     )
+    if not bento_obj.upload_status:
+        upload_status = DEFAULT_UPLOAD_STATUS
+    else:
+        upload_status = UploadStatus(
+            status=UploadStatus.Status.Value(bento_obj.upload_status['status'])
```

Comment:
could you also include the other fields here? there are also `updated_at`, `percentage`, and `error_message`